### DISCUSSION TITLE
Enabling GitHub projects and bug labels ?

### DISCUSSION THREAD

**Discussion by teolemon** [juliansteenbakker/mobile_scanner#261] [Ideas]
How about enabling GitHub projects and bug labels, so that we can better organize all 84 issues, spot duplicates… ?
I'd be very keen to do bug gardening given the appropriate rights.
Also, would be very keen to video chat @juliansteenbakker (also with @g123k)
Pierre

**Comment by juliansteenbakker** [comment]
Those seem like good improvements. The problem right now is that i don't have that much time to work on all those things. I manage a few other plugins, and besides that have a full-time job. On top of that, I will be moving to a new house next month, so any help on managing this is greatly appreciated!

I have invited you to be a collaborator of this project so you can manage some tickets and such.

Right now I am working on refactoring and improving the iOS code base. After that I'm gonna look to the macOS code base. My main focus right now is to make this plugin more stable on a wider set of devices.

**Reply by teolemon** [reply]
- They now have a issue type setting to distinguish bugs from enhancements without polluting labels
- If possible, I'd like the possibility to create an associated projects, or edit rights to an associated project, so that we can have organized views of the issues (boards)
- Finally GitHub also introduced subissues, which opens the door to creating epics (eg Autofocus, iOS…) and hierarchizing issues.
- Also, we could create P0, P1 labels to spot real showstoppers.
